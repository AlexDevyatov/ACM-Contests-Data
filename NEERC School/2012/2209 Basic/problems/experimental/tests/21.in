222 90
16 69
90 86
66 28
38 77
31 74
35 10
32 57
36 77
5 65
46 38
21 8
20 11
6 12
6 29
32 1
48 58
57 15
46 12
72 1
22 36
50 3
4 59
27 69
60 64
4 65
83 38
86 64
18 37
35 41
42 82
86 88
76 88
63 77
39 71
20 9
53 80
46 32
49 71
84 54
81 72
48 30
20 44
44 80
65 64
88 47
88 82
1 57
80 30
38 6
6 37
69 63
54 36
7 5
19 81
89 86
39 72
36 73
22 36
15 31
3 19
38 32
66 56
15 51
54 6
73 81
78 36
53 36
53 39
74 33
64 23
45 86
90 4
35 84
28 6
9 17
1 62
15 56
18 54
59 72
30 8
51 34
53 39
24 14
35 83
5 87
63 9
49 46
52 13
24 52
76 34
38 69
64 24
70 60
36 32
52 42
44 77
80 73
88 25
1 54
29 1
84 74
28 80
34 64
44 11
73 7
65 46
27 64
31 67
32 87
67 15
81 58
55 3
44 78
89 50
76 43
77 39
23 41
69 25
39 64
35 18
51 76
57 34
57 9
56 76
63 68
54 81
42 89
70 59
16 43
76 22
52 39
52 51
67 42
49 58
40 21
23 30
18 87
36 15
87 88
78 19
57 11
44 1
51 66
18 31
80 60
24 81
14 77
75 38
73 34
58 34
53 42
86 63
90 4
27 16
44 90
72 6
58 45
34 65
56 20
71 9
42 64
53 59
2 8
22 66
39 14
12 64
84 4
17 73
34 48
15 21
35 13
82 89
25 67
49 13
7 24
57 28
53 10
41 74
55 3
49 62
76 50
42 53
54 36
6 14
76 75
51 25
58 32
79 57
88 32
69 60
3 77
74 87
87 63
70 75
22 1
78 36
5 50
80 54
23 72
58 46
75 76
16 74
30 35
26 53
27 28
20 11
68 29
85 69
62 14
70 8
18 47
80 56
59 36
8 65
61 25
57 15
74 67
70 84
73 41
38 49
74 33
27 33
4 0 3 2 2 4 2 1 1 0 4 0 1 4 7 3 2 3 2 1 2 2 2 3 3 1 2 1 2 5 1 5 1 4 4 8 1 2 2 0 2 5 1 4 1 6 2 2 1 1 2 5 6 4 1 3 2 2 2 1 1 2 5 3 3 2 3 2 1 4 2 2 2 7 2 4 3 1 1 2 2 2 1 2 1 4 3 1 2 2
