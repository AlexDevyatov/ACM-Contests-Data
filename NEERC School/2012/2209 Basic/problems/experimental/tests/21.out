16 90 66 38 74 35 32 77 5 46 21 11 6 29 1 48 15 46 1 36 3 4 27 64 65 83 86 18 35 42 86 76 63 39 20 53 46 71 54 72 30 44 44 65 47 82 57 30 6 37 63 36 7 81 86 72 36 36 15 19 32 56 15 54 81 36 53 53 74 23 86 90 84 6 17 1 15 18 59 30 34 53 14 35 87 63 46 52 52 76 69 24 70 32 52 77 73 25 1 29 74 80 34 11 7 46 64 67 87 67 58 55 78 89 43 77 41 25 39 18 51 57 9 76 68 54 89 59 16 22 52 52 42 49 21 30 87 15 88 19 11 44 51 31 80 24 14 75 34 58 42 63 4 16 44 6 45 34 56 71 42 53 8 66 14 64 84 17 48 15 35 82 67 13 24 28 53 41 3 62 50 42 36 14 75 25 32 79 32 60 3 74 63 70 22 36 5 54 23 46 76 74 30 26 27 11 68 85 62 70 47 56 36 65 61 15 74 70 73 38 74 33
